build/linux/Makefile.in

```diff
@@ -281,13 +281,13 @@ ifeq ($(python),yes)
 py: libastra.la
 	$(MKDIR) python/build
 	$(MKDIR) python/finalbuild
-	cd $(srcdir)/../../python; CPPFLAGS="${PYCPPFLAGS}" LDFLAGS="${PYLDFLAGS}" $(PYTHON) builder.py build --build-base=$(abs_top_builddir)/python/build install \
+	cd $(srcdir)/../../python; CXX="${CXX}" CC="${CXX}" CPPFLAGS="${PYCPPFLAGS}" LDFLAGS="${PYLDFLAGS}" $(PYTHON) builder.py build --build-base=$(abs_top_builddir)/python/build install \
```

Comment:
We could also parse the CC variable in `Makefile.in` and use that, but then a user would have to set both `CXX` and `CC` to make it work correctly. Is that a standard way of doing this? If so, that might be the most elegant solution. I still don't know why python even compiles with CC even if the source file is a C++ file. It does the linking with CXX.